smoke/IotEdgeQuickstart/details/IotedgedWindows.cs

```diff
@@ -132,7 +133,34 @@ public async Task Configure(string connectionString, string image, string hostna
             }
         }
 
-        public Task Start() => Task.CompletedTask; // Runtime starts automatically when it's installed
+        public async Task Start()
+        {
+            Console.WriteLine($"Current OS Version: {Environment.OSVersion}");
+
+            // Need to start up service explicitly as service is not started up automatically after configured in Windows 10 RS4
```

Comment:
> service is not started up automatically after configured in Windows 10 RS4

Have you confirmed this is true generally, and not just a specific condition on your machine? I was probably using RS4 when I originally wrote `IotedgedWindows`.

Reply:
Confirmed with Venkat and Mike; this is the behavior in RS4; but it should start up right after configure in RS5.  Changed the approach to check status and start only if it is not running.

Reply:
Ok. Did it also start automatically in RS3?

Reply:
It starts automatically in every release but crashes immediately because it can't open the docker named pipe. That in turn happens because the iotedge service doesn't have a dependency on the docker service, so it starts up before the docker service can start up and create that named pipe. That is on my plate to fix with at least the Moby runtime in RS5. (Fixing it for the DockerForWindows runtime is harder.)